Week04/Kth smallest element in bst.cpp

```diff
@@ -0,0 +1,23 @@
+//TC: O(n)
+//MC: O(n), n is the no of nodes in bst
+
+class Solution {
+public:
+    int kthSmallest(TreeNode* root, int k) {
+        vector<int> tmp(2);
```

Comment:
Just as I have mentioned it for thousand times already, do not use `tmp` as variable name. Use something meaningful.